coordinator/pom.xml

```diff
@@ -43,8 +43,8 @@
   <!-- Shared properties -->
   <properties>
     <!-- Compiler -->
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>17</maven.compiler.source>
```

Comment:
I don't think that we need to change this in order to make it run on JDK22
we can keep it built for JDK11, this way we reduce the risks of subtle behavior changes and also it won't change the developer enviroment setup for existing developers

WDYT ?